pygmt/src/grdview.py

```diff
@@ -51,58 +50,55 @@ def grdview(self, grid, **kwargs):
+        modifier, and the projection is not oblique, the frontal facade between the
+        plane and the data perimeter is colored.
     surftype : str
-        Specify cover type of the grid.
-        Select one of following settings:
+        Specify cover type of the grid. Select one of following settings:
 
         - **m**: mesh plot [Default].
         - **mx** or **my**: waterfall plots (row or column profiles).
-        - **s**: surface plot, and optionally append **m** to have mesh lines
-          drawn on top of the surface.
+        - **s**: surface plot, and optionally append **m** to have mesh lines drawn on
+          top of the surface.
         - **i**: image plot.
         - **c**: Same as **i** but will make nodes with z = NaN transparent.
 
-        For any of these choices, you may force a monochrome image by
-        appending the modifier **+m**.
+        For any of these choices, you may force a monochrome image by appending the
+        modifier **+m**.
     contourpen : str
-        Draw contour lines on top of surface or mesh (not image). Append
-        pen attributes used for the contours.
+        Draw contour lines on top of surface or mesh (not image). Append pen attributes
+        used for the contours.
     meshpen : str
-        Set the pen attributes used for the mesh. You must also select
-        ``surftype`` of **m** or **sm** for meshlines to be drawn.
+        Set the pen attributes used for the mesh. You must also select ``surftype`` of
+        **m** or **sm** for meshlines to be drawn.
     facadepen :str
-        Set the pen attributes used for the facade. You must also select
-        ``plane`` for the facade outline to be drawn.
+        Set the pen attributes used for the facade. You must also select ``plane`` for
+        the facade outline to be drawn.
     shading : str
-        Provide the name of a grid file with intensities in the (-1,+1)
-        range, or a constant intensity to apply everywhere (affects the
-        ambient light). Alternatively, derive an intensity grid from the
-        input data grid reliefgrid via a call to :func:`pygmt.grdgradient`;
-        append **+a**\ *azimuth*, **+n**\ *args*, and **+m**\ *ambient* to
-        specify azimuth, intensity, and ambient arguments for that function,
-        or just give **+d** to select the default arguments
-        [Default is **+a**\ -45\ **+nt**\ 1\ **+m**\ 0].
+        Provide the name of a grid file with intensities in the (-1,+1) range, or a
+        constant intensity to apply everywhere (affects the ambient light).
+        Alternatively, derive an intensity grid from the input data grid reliefgrid via
+        a call to :func:`pygmt.grdgradient`; append **+a**\ *azimuth*, **+n**\ *args*,
+        and **+m**\ *ambient* to specify azimuth, intensity, and ambient arguments for
+        that function, or just give **+d** to select the default arguments [Default is
+        **+a**\ -45\ **+nt**\ 1\ **+m**\ 0].
```

Comment:
Does this string makes more sense here?
```suggestion
        ``"+a-45+nt1+m0"``].
```

Reply:
Yes, I think this makes sense, and I was also a bit thinking how to improve this part of the docstrings to make it clearer. Please see commit https://github.com/GenericMappingTools/pygmt/pull/3824/commits/f816e379156fb8c90f8bee78b5a74d145dc68184 for this change.